.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: dart
 dart:
-  - "1.17.1"
+  - "1.19.1"
```

Comment:
There are two more spots where the analyzer is failing on 1.20.1, but they're easy fixes:

_test/over_react/component_declaration/flux_component_test/handler_precedence.dart L21-25_
```diff
  @override
  getStoreHandlers() => {props.store.store1: increment};

- increment(_) {
+ increment(Store store) {
    numberOfHandlerCalls += 1;
  }
```

Same thing for _test/over_react/component_declaration/flux_component_test/store_handlers.dart L16-21_.

With those two changes, I think you'd be fine to bump this up to 1.20.1